packages/plugins/rubygem-ovirt_provision_plugin/rubygem-ovirt_provision_plugin.spec

```diff
@@ -18,7 +18,7 @@
 Summary:    Foreman plugin to provision new hosts and integrate to oVirt Engine
 Name:       %{?scl_prefix}rubygem-%{gem_name}
 Version:    2.0.3
-Release:    1%{?foremandist}%{?dist}
+Release:    2%{?foremandist}%{?dist}
```

Comment:
This also needs a changelog entry below, and i'm not sure if it can be done in the same pr or not